CoreService/Trustees.psm1

```diff
@@ -90,6 +90,53 @@ function Get-User
 	}
 }
 
+function Get-Group
+{
+    <#
+    .Synopsis
+    Gets information about the a specific Tridion group.
+
+    .Description
+    Gets a group object containing information about the specified group within Tridion.
+
+    .Inputs
+    None.
+
+    .Outputs
+    Returns an object of type [Tridion.ContentManager.CoreService.Client.TrusteeData].
+
+    .Link
+    Get the latest version of this script from the following URL:
+    https://github.com/pkjaer/tridion-powershell-modules
+
+    .Example
+    Get-TridionGroup "WebMaster"
+    Returns information about the group 'WebMaster'.
+	
+	.Example
+    Get-TridionUser "tcm:2-12-65552"
+    Returns information about group with the id 'tcm:2-12-65552'.
+    
+    #>
+    [CmdletBinding()]
+    Param
+    (
+        [Parameter(ValueFromPipelineByPropertyName=$true)]
+		[ValidateNotNullOrEmpty()]
+        [string]$Group
+    )
+
+	Write-Verbose "Loading Tridion group...";
+	$GroupObj = Get-Groups | ?{($group -eq ($_.Id)) -or ($group -eq $_.Title)} | Select -First 1
```

Comment:
Do you think it should be a parameter set?
What do you say about the option that the "group" may be either a tcm id OR it's name (title)?
A check of whether it's an ID or title is easy. Just do $Group.StartsWith('tcm').
I think this kind of usage will be easier to use.

Of course I'll also run Get-Groups only if it's a title :)